src/entities/MovieDetails/MovieDetails.jsx

```diff
@@ -1,9 +1,50 @@
 import "./MovieDetails.css";
 
-export const MovieDetails = ({ movie, handleMovieViewClosed }) => {
+import {
+  useLoaderData,
+  useSearchParams,
+  createSearchParams,
+  useNavigate,
+} from "react-router-dom";
 
-  const { imgUrl, movieName, releaseYear, rating, duration, description } =
-    movie;
+export const MovieDetails = ({
+  movie = {
+    movieName: "La La Land",
+    releaseYear: 2016 - 12 - 29,
+    imgUrl: "https://via.placeholder.com/300/92c952",
+    duration: "128",
+    rating: 7.9,
+    description:
+      "Mia, an aspiring actress, serves lattes to movie stars in between auditions and Sebastian, a jazz musician, scrapes by playing cocktail party gigs in dingy bars, but as success mounts they are faced with decisions that begin to fray the fragile fabric of their love affair, and the dreams they worked so hard to maintain in each other threaten to rip them apart.",
+  },
+}) => {
+  const [searchParams] = useSearchParams();
+  const navigate = useNavigate();
+
+  let imgUrl, movieName, releaseYear, rating, duration, description;
+
+  const data = useLoaderData();
+
+  if (data.message === "error") {
```

Comment:
really strange approach to take data from the error  :)

Reply:
in our case backend endpoint doesn't work, so I have tried to put a mock data inside error part

Reply:
yeah, that's fine for our case